### ISSUE TITLE
GDAL Error 1: PROJ: proj_identify: Cannot find proj.db

### ISSUE THREAD

**Issue by anastassiavybornova** [anastassiavybornova/bike-node-planner-data-denmark#31]
won't fix for now because the only problem is the error message when running the bash script - workflow works as expected nevertheless.

xrefs: #24  & https://github.com/rspatial/terra/issues/1555

**Comment by anerv** [comment]
If it works without, we can just remove it - then it's also one less setting for the user to configure.

However, I just did a bit of googling and as far as I can see it is GDAL, not fiona, that is complaining?
So I just tried one more time using a non-fiona proj path, and now it works without warnings: `/Applications/QGIS-LTR.app/Contents/Resources/proj/`

I've reinstalled QGIS since last time I tried so this might be why - but if it also works for you, it might make sense to update the instructions to ask for the general proj.db path, not fiona - and then add to ignore potential warnings, in case it does not work? I'll be happy to do the update if you agree.
